frontend/src/components/PlantDetailsView/PlantDetailsView.tsx

```diff
@@ -1,11 +1,18 @@
 import React, { useState } from 'react'
-import { Card, Col, Row } from 'react-bootstrap';
+import { Button, Card, Col, Row } from 'react-bootstrap';
 import { useNavigate, useParams } from 'react-router-dom'
 import { mockPlants } from '../../utils/mockData';
 import Calendar from '../Calendar/Calendar';
 import { DetailsWrapper, InfoWrapper, TitleSeparator } from './PlantDetailsViewStyles';
 import { ContentContainer } from "../App/AppStyle";
 
+
+function edit_plant(){}
```

Comment:
Wszędzie używamy camel case'a, nawet w placeholder'owych funkcjach to kłuje w oczy.

Reply:
naprawione (następny commit, czyli c27ee7e86b4e46f71723bc0d91d9295bcbfe606a)